RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress+View.swift

```diff
@@ -0,0 +1,174 @@
+extension AccountRecoveryScanInProgress.State {
+	var viewState: AccountRecoveryScanInProgress.ViewState {
+		.init(
+			status: status,
+			kind: factorSourceIDFromHash.kind,
+			olympia: forOlympiaAccounts,
+			active: active,
+			hasFoundAnyAccounts: !active.isEmpty || !inactive.isEmpty,
+			maxIndex: batchNumber * batchSize
+		)
+	}
+}
+
+// MARK: - AccountRecoveryScanInProgress.View
+public let batchSize = 50
```

Comment:
cannot be `private` or `fileprivate` since used on both the View and the Reducer. And since we are... mono repo! `internal` vs `public` does in fact not make any difference. But sure, I will make it `internal` (drop `public`).

Reply:
fixed in https://github.com/radixdlt/babylon-wallet-ios/pull/941